### ISSUE TITLE
Add Demo deployment

### ISSUE THREAD

**Issue by unstephenk** [ushahidi/platform-android#16]
The earlier version of the application has a demo url for deployment testing. This should be included in the newest  version

**Comment by eyedol** [comment]
Agreed. Packing this on the backlog as it will require a bit of work on platform side.
